app/models/vote.rb

```diff
@@ -0,0 +1,5 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
+
```

Comment:
If you wanted to make it so that each user can only vote for a given work once, you could do it with a validation here.